lib/seattleflu/db/cli/command/preprocess.py

```diff
@@ -141,8 +141,9 @@ def load_uw_metadata(uw_filename: str) -> pd.DataFrame:
     Given a filename *uw_filename*, returns a pandas DataFrame containing
     clinical metadata.
     """
+    na_values = ['Unknown']
```

Comment:
Is it not necessary to also name `NULL` here, since it was in the previous value maps in the etl routine?

Reply:
Nope. So long as the default na_values are kept in pandas (keep_default_na=True (by default)), 'NULL' will be converted to null.

Reply:
Ah, thanks!